app/models/article.rb

```diff
@@ -497,12 +504,15 @@ def validate_tag
 
   def remove_tag_adjustments_from_tag_list
     tags_to_remove = TagAdjustment.where(article_id: id, adjustment_type: "removal", status: "committed").pluck(:tag_name)
-    tag_list.remove(tags_to_remove, parser: ActsAsTaggableOn::TagParser) if tags_to_remove.present?
+    tag_list.remove(tags_to_remove, parse: true) if tags_to_remove.present?
```

Comment:
@maestromac what is "parse: true" ?

Reply:
previously with `parser: ActsAsTaggableOn::TagParser` we ask the tags string (or array) to be parsed with a particular parser. With `parse: true`, we ask it to use the default parser.